packages/design-system/src/components/SearchBar/SearchBar.js

```diff
@@ -268,14 +280,15 @@ SearchBar.propTypes = {
   onCollapse: PropTypes.func,
   onExpand: PropTypes.func,
   onFocus: PropTypes.func,
-  onBlur: PropTypes.func
+  onBlur: PropTypes.func,
+  forwardedRef: PropTypes.func
 };
 
 SearchBar.defaultProps = {
   placeholder: 'Search...',
   debounceTime: 0
 };
 
-export default React.forwardRef(function SearchBarWithRef(props, ref) {
```

Comment:
if you just use an anonymous arrow function here the component will be harder to find in React Dev Tools (https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools) + possible error message will be less helpful